homeassistant/components/tibber/config_flow.py

```diff
@@ -44,10 +44,14 @@ async def async_step_user(
                 await tibber_connection.update_info()
             except asyncio.TimeoutError:
                 errors[CONF_ACCESS_TOKEN] = "timeout"
-            except aiohttp.ClientError:
-                errors[CONF_ACCESS_TOKEN] = "cannot_connect"
             except tibber.InvalidLogin:
                 errors[CONF_ACCESS_TOKEN] = "invalid_access_token"
+            except (
```

Comment:
We normally require 100% coverage on config flows. Could tests be added?

Reply:
I have near to 0 experience on python/unit testing on python and thus i've been strugling with this repo we try to help migrate to the new websocket endpoint. All assistance is welcome! @epenet.

Reply:
I will have a look, shall I just create a new PR when done?

Reply:
Yes please - and add a comment here with the link to the new PR

Reply:
#89088 adds unit tests for the config flow errors